core/helpers/hardhat.ts

```diff
@@ -76,8 +81,14 @@ export const createWalletForRpc = async (signerPrivateKey: string, network: stri
     return hre.network.provider;
 };
 
+export const warpTime = async function (network: string, blocks = 20000, secondsBetweenBlocks = 270) {
+    const provider = await getProvider(network);
+    await provider.send('hardhat_mine', [`0x${blocks.toString(16)}`, `0x${secondsBetweenBlocks.toString(16)}`]);
```

Comment:
Don't you have `formatToHex` helper for this? Can you please use this helper everywhere, including `network.ts`, `rpc.ts`, `surplus-test.ts`?

Reply:
`network.ts` does not have anything to use this on as i see.